charts/parca/values.yaml

```diff
@@ -135,6 +135,9 @@ server:
   tolerations: {}
   # -- resource limits and requests for server pod
   resources: {}
+  # -- clusterrole and binding for kubernetes service discovery mechanism
+  rbac:
+    create: true
```

Comment:
are there actually still clusters out there where you don't have RBAC? Since we default to true I don't really care just an observation...